checkpoint_connector.py

```diff
@@ -178,11 +177,20 @@ def _make_rest_call(self, endpoint, body, action_result):
 
         return phantom.APP_SUCCESS, resp_json
 
-    def _login(self, action_result):
+    def _set_auth_sid(self, action_result):
+        if not self._state.get('sid'):
```

Comment:
can `sid` be 0? if so that case would get incorrectly caught in this `if`

not sure if it's passed around strictly as a string in which case the above point is moot

Reply:
sid cannot be 0 and is always a string